lib/getConnectors/index.js

```diff
@@ -21,106 +20,99 @@ module.exports = function (directory) {
 
 	connectors = _.map(connectorDirectories, function (connectorDirectory) {
 
-		var connector;
+		var connector,
+			fullConnectorPath = directory + '/connectors/' + connectorDirectory;
 
 		// Require the connector file
-		var result = getFiles(directory+'/connectors/'+connectorDirectory);
-		if (result.indexOf('connector.js') !== -1) {
-			connector = require(directory+'/connectors/'+connectorDirectory+'/connector');
-		} else {
-			connector = {};
+		try {
```

Comment:
That first try/catch is separate because `connector.name` does not exist at that point in code (it's declared later), and so the associated error message had to be different